server/@types/CvlUserDetails.ts

```diff
@@ -5,6 +5,9 @@ export default class CvlUserDetails {
   // e.g. smith
   lastName?: string
 
+  // e.g. John_Smith
+  username?: string
```

Comment:
I can't see where this is being set on the CVL User Details type?

Reply:
removed the username for cvluserdetail class